### PR TITLE
add aliyun oss example

### PR DESCRIPTION
**Pull request by pepezzzz**
### First-time contributors' checklist 

- [ x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add aliyun oss example

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?
https://help.aliyun.com/document_detail/185803.html

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lichunzhu — APPROVED** [review]
LGTM

**Review by 3pointer — APPROVED** [review]

**Review by lichunzhu — APPROVED** [review]
LGTM

File: br/backup-and-restore-storages.md [hunk 1]
```diff
@@ -157,6 +157,43 @@ S3、 GCS 和 Azblob 等云存储有时需要额外的连接配置，你可以
 | `--s3.acl` | 上传对象的 canned ACL（例如，`private`、`authenticated-read`） |
 | `--s3.provider` | S3 兼容服务类型（支持 `aws`、`alibaba`、`ceph`、`netease` 或 `other`） |
 
+导出到非 AWS 的 S3 云服务时，需要指定云服务商名字，对应是否使用 virtual hosted style。以阿里云的 OSS 存储为例：
```
Comment (TomShawn) [inline br/backup-and-restore-storages.md:160]:
```suggestion
将数据导出到非 AWS 的 S3 云服务时，你需要指定云服务商名字，对应是否使用 virtual hosted style。以阿里云的 OSS 存储为例：
```

File: br/backup-and-restore-storages.md [hunk 2]
```diff
@@ -157,6 +157,43 @@ S3、 GCS 和 Azblob 等云存储有时需要额外的连接配置，你可以
...
+
+* Dumpling 使用 OSS 存储
```
Comment (TomShawn) [inline br/backup-and-restore-storages.md:162]:
```suggestion
* 使用 Dumpling 将数据导出至 OSS 存储：
```

File: br/backup-and-restore-storages.md [hunk 3]
```diff
@@ -157,6 +157,43 @@ S3、 GCS 和 Azblob 等云存储有时需要额外的连接配置，你可以
...
+
+```bash
+./dumpling -h 127.0.0.1 -P 3306 -B mydb -F 256MiB \
+   -o "s3://my-bucket/dumpling/" \
+   --s3.endpoint="http://oss-cn-hangzhou-internal.aliyuncs.com" \
+   --s3.provider="alibaba" \
+   -r 200000 -F 256MiB
+```
+
+* BR 使用 OSS 存储
```
Comment (TomShawn) [inline br/backup-and-restore-storages.md:174]:
```suggestion
* 使用 BR 将数据备份至 OSS 存储：
```

File: br/backup-and-restore-storages.md [hunk 4]
```diff
@@ -157,6 +157,43 @@ S3、 GCS 和 Azblob 等云存储有时需要额外的连接配置，你可以
...
+./br backup full --pd "127.0.0.1:2379" \
+    --storage "s3://my-bucket/full/" \
+    --s3.endpoint="http://oss-cn-hangzhou-internal.aliyuncs.com" \
+    --s3.provider="alibaba" \
+    --send-credentials-to-tikv=true \
+    --ratelimit 128 \
+    --log-file backuptable.log
+```
+
+* Lightning 使用 OSS 存储的 Yaml 文件
```
Comment (shichun-0415) [inline br/backup-and-restore-storages.md:188]:
```suggestion
* TiDB Lightning 使用 OSS 存储的 Yaml 文件
```

Comment (TomShawn) [inline br/backup-and-restore-storages.md:188]:
```suggestion
* 在 YAML 文件中指定 TiDB Lightning 将数据导出至 OSS 存储：
```

File: br/backup-and-restore-storages.md [hunk 5]
```diff
@@ -157,6 +157,43 @@ S3、 GCS 和 Azblob 等云存储有时需要额外的连接配置，你可以
...
+
+{{< copyable "yaml" >}}
+
+```yaml
+[mydumper]
+data-source-dir = "s3://my-bucket/dumpling/?endpoint=http://oss-cn-hangzhou-internal.aliyuncs.com&provider=alibaba"
```
Comment (shichun-0415) [inline br/backup-and-restore-storages.md:194]: my-bucket/dumpling/-----Why is dumpling in the link? Should it be lightning?

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**